CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.1.0-beta2
+#### Added
+- Support for `action://your-custom-action-name` url scheme for calling custom actions and reserved `itbl://sdk-custom-action` scheme for SDK internal actions. 
+	- For example, if you want a custom action called `buyCoffee` to be called via `IterableCustomActionDelegate` when user taps on the 'Buy' button in your in-app, you have to set the href of the button as `action://buyCoffee`.
```

Comment:
```suggestion
	- For example, to have `IterableCustomActionDelegate` call a custom `buyCoffee` action when a user taps on an in-app message's **Buy** button, set the href of the button to `action://buyCoffee`.
```

How is the button URL labeled when setting up the in-app? Is it **href** or is it **URL**? Would probably change this sentence to match if necessary.

I'd also put this underneath the appropriate bullet above (since I separated them)

Reply:
It is like any other regular html button so it is **href**.